@navikt/ds-css/layout/layout.css

```diff
@@ -0,0 +1,81 @@
+.navds-layout-container {
```

Comment:
For BEM så vil det være mer riktig å skrive `.navds-layout__container` her?

Reply:
Fiksa :)